src/unity/paths/operator_accounts_accountId.yml

```diff
@@ -0,0 +1,87 @@
+# Oats is making this incredibly annoying with naming conflicts
+get:
+  operationId: GetAccountById
+  tags:
+    - Accounts
+    - Operator
+  parameters:
+    - $ref: '../../common/parameters/TraceSpan.yml'
+    - in: path
+      name: accountId
+      schema:
+        type: string
+      required: true
+      description: The ID of the account to get.
+  responses:
+    '200':
+      description: the account
+      content:
+        application/json:
+          schema:
+              allOf:
+                - $ref: '../schemas/Account.yml'
+                - properties:
```

Comment:
the operator get account returns an account with layers of nested values nested values
account
  - billing contact
  - users
  - marketplace subscription
  - organizations
    - each org has its account nested

Reply:
I might be speaking too soon since I haven't gone through all the changes, but I think we currently have a `RelatedAccount` prop that's a limited version of the org's account so that we don't have a circular dependency of `orgs <-> account`. Is that still the case?

**UPDATE:** Just saw that the account was updated to remove the organization <3